docs/content/en/docs/migrate/evolution-klt/_index.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Evolution of KLT
+description: Understand the paradigm of KLT, how it evolved from Keptn v1, and whether you should migrate to KLT
+weight: 5
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+The Keptn products provide tool-agnostic ways
+to manage and observe software deployment.
+Keptn connects to the tools you choose
+through an open event standard
+to automate delivery and remediation sequences.
+
+Two different products are available:
+
+* **First Generation:** Keptn v1.y.z LTS, referred to as "Keptn v1"
```

Comment:
They aren't first and second generation. They are two different projects - [according to our historical stance and the documentation as it still stands](https://github.com/keptn/lifecycle-toolkit/blob/53443ac4042e5f78e402df7378770fd163b5167c/docs/content/en/docs/intro-klt.md?plain=1#L66).

Reply:
I changed it to two differe sub-projects, at least for now.  Again, we'll get more eyes on this before it's published...